llvm/lib/MCCAS/MCCASObjectV1.cpp

```diff
@@ -1979,21 +1989,20 @@ struct DIEToCASConverter {
               SmallVectorImpl<std::unique_ptr<DIEDataWriter>> &DIEWriters);
 };
 
-Error InMemoryCASDWARFObject::partitionCUData(ArrayRef<char> DebugInfoData,
-                                              uint64_t AbbrevOffset,
-                                              DWARFContext *Ctx,
-                                              MCCASBuilder &Builder,
-                                              AbbrevSetWriter &AbbrevWriter,
-                                              uint16_t DwarfVersion) {
+Error InMemoryCASDWARFObject::partitionCUData(
+    ArrayRef<char> DebugInfoData, uint64_t AbbrevOffset, DWARFContext *Ctx,
+    MCCASBuilder &Builder, AbbrevSetWriter &AbbrevWriter, uint16_t DwarfVersion,
```

Comment:
If we have a MCCASBuilder, we have a MachOCASWriter, which has a Target; we can add a method `MCCASBuilder::getAddressSize` and avoid the needed for the redundant `AddressSize` parameter